internal/engine/wazevo/backend/isa/arm64/instr.go

```diff
@@ -819,6 +869,22 @@ func (i *instruction) asVecShiftImm(op vecOp, rd, rn, rm operand, arr vecArrange
 	i.u2 = uint64(arr)
 }
 
+func (i *instruction) asVecTbl(nregs byte, rd, rn, rm operand, arr vecArrangement) {
+	switch nregs {
+	case 0, 1:
+		i.kind = vecTbl
+	case 2:
+		i.kind = vecTbl2
+		if !rn.reg().IsRealReg() {
+			panic("rn is not a RealReg")
+		}
```

Comment:
I think also we should have the assertion that rn!=v31 (which makes rn+1 invalid)